uaparser/parsing_test.go

```diff
@@ -38,8 +44,13 @@ func TestOSParsing(t *testing.T) {
 	}
 }
 
-func TestReadsInteralYAML(t *testing.T) {
-	_ = NewFromSaved() // should not panic
+func TestReadsInternalYAML(t *testing.T) {
+	t.Parallel()
+
+	_, err := New(DefinitionYaml)
```

Comment:
this looks like a bug?
```suggestion
	_, err := New()
```

seems there are probably other compile errors as well